### ISSUE TITLE
Running >v1.65 error git failed. Is it installed, and are you in a Git repository directory?

### ISSUE THREAD

**Issue by diegolagospagopa** [antonbabenko/pre-commit-terraform#370]
<!--
Thank you for helping to improve pre-commit-terraform!

Please be sure to search for open issues before raising a new one. We use issues
for bug reports and feature requests. Please note, this template is for bugs
report, not feature requests.
-->

### Describe the bug

Using this configuration:

```yaml
repos:
  - repo: https://github.com/antonbabenko/pre-commit-terraform
    rev: v1.64.0
    hooks:
      - id: terraform_fmt
      - id: terraform_docs
      # - id: terraform_tfsec
      #   args:
      #     - --args=--exclude-downloaded-modules
      - id: terraform_validate
```

running this command:

```sh
docker run -v $(pwd):/lint -it  -w /lint ghcr.io/antonbabenko/pre-commit-terraform:v1.68.0 run -a
```

i have this error

```sh
An error has occurred: FatalError: git failed. Is it installed, and are you in a Git repository directory?
Check the log at /root/.cache/pre-commit/pre-commit.log
```

#### Notes

The error arise from the version v1.65, the v1.64 doesn't have this problem. And this problem is present even in the github actions with ubuntu:20.04 [azurerm action log](https://github.com/pagopa/azurerm/runs/6076472627?check_suite_focus=true)


### How can we reproduce it?

run this command

```sh
docker run -v $(pwd):/lint -it  -w /lint ghcr.io/antonbabenko/pre-commit-terraform:v1.68.0 run -a
```

On this repo: [pagopa azurerm](https://github.com/pagopa/azurerm) into the root folder

### Environment information

* OS:  

OS: MacOS
OS: Ubuntu 20.04 (github action)

* `docker info`:

<details><summary><code>docker info</summary>

```bash
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  compose: Docker Compose (Docker Inc., v2.2.3)
WARNING: Plugin "/Users/diego/.docker/cli-plugins/docker-buildx" is not valid: failed to fetch metadata: fork/exec /Users/diego/.docker/cli-plugins/docker-buildx: no such file or directory

Server:
 Containers: 32
  Running: 0
  Paused: 0
  Stopped: 32
 Images: 11
 Server Version: 20.10.11
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runtime.v1.linux runc io.containerd.runc.v2
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 1e5ef943eb76627a6d3b6de8cd1ef6537f393a71
 runc version: b9ee9c6314599f1b4a7f497e1f1f856fe433d3b7
 init version: 
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.10.103-0-virt
 Operating System: Alpine Linux v3.14
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 2.866GiB
 Name: lima-rancher-desktop
 ID: ERTZ:BKSE:Z53B:JTZH:GPJL:LLPO:AW7N:W44Y:BMPX:4KK3:W5D4:TY7Z
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

</details>

* Docker image tag/git commit:  v.1.68 and nightly

* Error output

```sh
An error has occurred: FatalError: git failed. Is it installed, and are you in a Git repository directory?
Check the log at /root/.cache/pre-commit/pre-commit.log
```

* `.pre-commit-config.yaml`:

<details><summary>.pre-commit-config.yaml</summary>

```bash
repos:
  - repo: https://github.com/antonbabenko/pre-commit-terraform
    rev: v1.64.0
    hooks:
      - id: terraform_fmt
      - id: terraform_docs
      # - id: terraform_tfsec
      #   args:
      #     - --args=--exclude-downloaded-modules
      - id: terraform_validate

```
</details>

* log `/root/.cache/pre-commit/pre-commit.log`

<details><summary>/root/.cache/pre-commit/pre-commit.log</summary>
```sh

  version information                                                                                                      
                                                                                                                             
                                                                                                                        
  pre-commit version: 2.18.1                                                                                                   
  git --version: git version 2.34.2                                                                                            
  sys.version:                                                                                                                 
      3.10.1 (main, Dec  8 2021, 04:32:13) [GCC 10.3.1 20211027]                                                               
  sys.executable: /usr/local/bin/python3                                                                                       
  os.name: posix                                                                                                               
  sys.platform: linux                                                                                                          
                                                                                                                         
                                                                                                                               
   error information                                                                                                        
                                                                                                                                 ```                                                                                                                          
  An error has occurred: FatalError: git failed. Is it installed, and are you in a Git repository directory?                   
                                                                                                                         
                                                                                                                               
                                                                                                                     
  Traceback (most recent call last):                                                                                           
    File "/usr/local/lib/python3.10/site-packages/pre_commit/git.py", line 58, in get_root                                     
      cmd_output('git', 'rev-parse', '--show-cdup')[1].strip(),                                                                
    File "/usr/local/lib/python3.10/site-packages/pre_commit/util.py", line 152, in cmd_output                                 
      returncode, stdout_b, stderr_b = cmd_output_b(*cmd, **kwargs)                                                            
    File "/usr/local/lib/python3.10/site-packages/pre_commit/util.py", line 146, in cmd_output_b                               
      raise CalledProcessError(returncode, cmd, retcode, stdout_b, stderr_b)                                                   
  pre_commit.util.CalledProcessError: command: ('/usr/bin/git', 'rev-parse', '--show-cdup')                                    
  return code: 128                                                                                                             
  expected return code: 0                                                                                                      
  stdout: (none)                                                                                                               
  stderr:                                                                                                                      
      fatal: unsafe repository ('/lint' is owned by someone else)                                                              
      To add an exception for this directory, call:                                                                            
                                                                                                                               
          git config --global --add safe.directory /lint
  
  During handling of the above exception, another exception occurred:                                                          
                                                                                                                               
  Traceback (most recent call last):                                                                                           
    File "/usr/local/lib/python3.10/site-packages/pre_commit/error_handler.py", line 73, in error_handler                      
      yield                                                                                                                    
    File "/usr/local/lib/python3.10/site-packages/pre_commit/main.py", line 326, in main                                       
      _adjust_args_and_chdir(args)                                                                                             
    File "/usr/local/lib/python3.10/site-packages/pre_commit/main.py", line 143, in _adjust_args_and_chdir                     
      toplevel = git.get_root()                                                                                                
    File "/usr/local/lib/python3.10/site-packages/pre_commit/git.py", line 64, in get_root                                     
      raise FatalError(                                                                                                        
  pre_commit.errors.FatalError: git failed. Is it installed, and are you in a Git repository directory?                        
                                                                                                                          
  - /root/.cache/pre-commit/pre-commit.log 50/50 100%

```

</details>

**Comment by MaxymVlasov** [comment]
Confirmed.

Works:
```bash
docker run -v $(pwd):/lint -it  -w /lint ghcr.io/antonbabenko/pre-commit-terraform:v1.64.1 run -a
```

Fail:
```bash
docker run -v $(pwd):/lint -it  -w /lint ghcr.io/antonbabenko/pre-commit-terraform:v1.65.0 run -a
```

**Comment by MaxymVlasov** [comment]
1.64.1 released 31 Mar. 1.65.0 - 13 Apr.

[No changes in Dockerfile](https://github.com/antonbabenko/pre-commit-terraform/compare/v1.64.1...v1.65.0) => issue with base image/packages

**Comment by diegolagospagopa** [comment]
> 1.64.1 released 31 Mar. 1.65.0 - 13 Apr.
> 
> [No changes in Dockerfile](https://github.com/antonbabenko/pre-commit-terraform/compare/v1.64.1...v1.65.0) => issue with base image/packages

Ok great, if i can help please let me know

**Comment by MaxymVlasov** [comment]
Use v1.64.1 for now. I'll write here when it will be fixed

**Comment by diegolagospagopa** [comment]
Ok good ;D

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.68.1 :tada:

**Comment by umesh-sha-mali** [comment]
@antonbabenko we have faced this in v1.76.0 version too..
